### PR TITLE
feat(runner): support multiple names in project filter

### LINKED ISSUE #8583: [Feature] Allow running multiple (but not all) projects via commandline
👋   Currently there is an option to run either a single project or all of them. It's very strict. Sometimes you need a combination of projects so it'd be nice if there would be an option to pass f.e. a comma-separated string of values to run multiple projects.

Example:

```
// playwright.config.ts
import { PlaywrightTestConfig } from '@playwright/test';
import { TestOptions } from './my-test';

const config: PlaywrightTestConfig<TestOptions> = {
  timeout: 20000,
  projects: [
    {
      name: 'v1',
      use: { version: '1.0' },
    },
    {
      name: 'v2',
      use: { version: '2.0' },
    },
    {
      name: 'v3',
      use: { version: '3.0' },
    },
    {
      name: 'v4',
      use: { version: '4.0' },
    },
  ]
};
export default config;
```

`npx playwright test --project="v2,v4"`

### PR DESCRIPTION
**Pull request by yury-s**
Fixes #8583

### CODE REVIEW

File: src/test/runner.ts [hunk 1]
```diff
@@ -140,14 +140,23 @@ export class Runner {
...
+    if (projectName) {
+      projectsToFind = new Map();
+      projectName.split(',').forEach(n => {
+        const name = n.trim();
+        projectsToFind!.set(name.toLocaleLowerCase(), name);
+      });
+    }
     const projects = this._loader.projects().filter(project => {
-      return !projectName || project.config.name.toLocaleLowerCase() === projectName.toLocaleLowerCase();
+      return !projectsToFind || projectsToFind.delete(project.config.name.toLocaleLowerCase());
```
Comment (dgozman) [inline src/test/runner.ts:152]: This breaks when you have multiple projects with the same name. Our tests do that :)

File: src/test/cli.ts [hunk 2]
```diff
@@ -57,7 +57,7 @@ export function addTestCommand(program: commander.CommanderStatic) {
   command.option('--reporter <reporter>', `Reporter to use, comma-separated, can be ${builtInReporters.map(name => `"${name}"`).join(', ')} (default: "${defaultReporter}")`);
   command.option('--retries <retries>', `Maximum retry count for flaky tests, zero for no retries (default: no retries)`);
   command.option('--shard <shard>', `Shard tests and execute only the selected shard, specify in the form "current/all", 1-based, for example "3/5"`);
-  command.option('--project <project-name>', `Only run tests from the specified project (default: run all projects)`);
+  command.option('--project <project-name>[,<project-name>...]', `Only run tests from the specified coma-separated list of projects (default: run all projects)`);
```
Comment (dgozman) [inline src/test/cli.ts:60]: typo: coma

**Review by dgozman — APPROVED** [review]

File: src/test/cli.ts (hunk 2 above)
Comment (yury-s) [reply src/test/cli.ts:60]: done

File: src/test/runner.ts (hunk 1 above)
Comment (yury-s) [reply src/test/runner.ts:152]: This should be address by #8604 which prohibits duplicate project names.

**Pull request merged**